website/guides/03_git-guide.mdx

```diff
@@ -0,0 +1,425 @@
+---
+title: The guide to Git I never had.
+slug: git
+description: 'Install ingress-nginx for Kubernetes with Glasskube package manager including a GUI, CLI and automatic package updates'
```

Comment:
This description is important for Google. Maybe we can highlight the inidvidual commands that are explained - The description should not exceed 160 chars.

Reply:
Done!